src/EditorServicesCommandSuite/CodeGeneration/PowerShellScriptWriter.cs

```diff
@@ -468,6 +470,17 @@ internal void WriteHashtableEntry(string key, Action valueWriter)
             valueWriter();
         }
 
+        internal void WriteHashtableEntry(string key, Action valueWriter, int aligner)
```

Comment:
Maybe you are right. I'm not ready to let this part go _just yet_. I am totally OK with making this optional, but I feel it is a useful and logical addition to the feature. 

- If I were using `ConvertTo-SplatExpression -AllParameters` I would have the intention of filling in quite a few of the listed params, and probably work my way through them line by line. It's very useful to have the equal signs aligned in that case. VScode can do that for me, but it's an extra step that i would always be performing. 
- You are already formatting the splat by indenting it, why not align the equal signs?

But: it is up to you in the end ;-) let me know what you think.

Reply:
That's totally understandable, let me explain my position in a bit more detail.

> You are already formatting the splat by indenting it, why not align the equal signs?

Alignment is a preference, one that's pretty split in the community.  Indentation on the other hand is mandatory.  I don't think there are a ton of folks who prefer no indentation.  It's also **way** easier to implement.  You only need to worry about white space, and preference can be controlled with a single `TabString` setting.

Alignment is a lot harder to control in a way that plays nice for everyone.

For example, what if the user prefers tabs?

- What if the target alignment isn't a multiple of four?  Do we insert tab characters until the remainder and then insert spaces?

- How do we know the actual length of their tab character?

There's also tons of characters that have a length greater than one but still only take up one visible space.  Emoji's for example (e.g. 😀 or `"`u{1f600}"` in PowerShell Core) have a length of two.  Eh.. I guess you could get around that by using `System.Globalization.StringInfo.LengthInTextElements`.  I wonder if that works the same in Unix based platforms.

Alright I'm slowly talking myself into it :).  If you can figure out a way to handle tab character preference uniformly, I'm in.  Otherwise I think we should hold off on alignment for now.

Reply:
I've seen the tabs vs. spaces discussions, just had not thought of them in this regard. I just happen to be a spaces person (but no tab-hater, lol), and as such, have not aligned things with `something\t\t\      = ...` much, and only thought of tabs vs. spaces in the indentation context. But yeah, you are right, and yes, that does make it more complex to implement. Let me ponder on this one a bit, ok?

Reply:
OK I've pondered on this a bit and I think it's not worth the effort. `Alt-Shift-F` is good enough.